src/prostredi/editor/clion/clion.md

```diff
@@ -0,0 +1,51 @@
+## Instalace CLionu
+
+Nejlepší způsob instalace je použití aplikace [Toolbox](https://www.jetbrains.com/toolbox-app/), která vám umožní spravovat všechna vaše IDE od JetBrains. Pokud narazíte na problém, kompletní návod
+naleznete [zde](https://www.jetbrains.com/help/clion/installation-guide.html#toolbox).
+
+## Licence
+
+CLion nabízí 30denní zkušební verzi (trial) pro vyzkoušení. Pokud chcete CLion používat po této době, můžete požádat o [školní licenci](https://www.jetbrains.com/shop/eform/students).
+
+- Na stránce jsou čtyři způsoby, jak licenci získat. Nejjednodušší je použít váš školní e-mail. Pokud jej neznáte, můžete jej najít vlevo nahoře v [Roundcube](https://posta.vsb.cz/roundcube/).
```

Comment:
Tady bych prostě napsal, že e-mail je `<login>@vsb.cz`, např. `UPR0123@vsb.cz`. Pokud neví svůj e-mail, tak ho v roundcube nenajdou :)

Reply:
V roundcube je ta delsi varianta toho emailu, ale tvuj zpusob je lepsi, takze opravim

Reply:
Fixed in https://github.com/mrlvsb/upr-skripta/pull/94/commits/2f0c883cbbe24ed863bc5e4e6a15d37da5839489